libs/extensions/std/lang/src/meta-inf-example-validation.spec.ts

```diff
@@ -0,0 +1,120 @@
+// SPDX-FileCopyrightText: 2023 Friedrich-Alexander-Universitat Erlangen-Nurnberg
+//
+// SPDX-License-Identifier: AGPL-3.0-only
+
+import {
+  BlockDefinition,
+  BlockMetaInformation,
+  JayveeServices,
+  createJayveeServices,
+  useExtension,
+} from '@jvalue/jayvee-language-server';
+import {
+  ParseHelperOptions,
+  TestLangExtension,
+  ValidationResult,
+  expectNoParserAndLexerErrors,
+  getTestExtensionBlockForIOType,
+  parseHelper,
+  validationHelper,
+} from '@jvalue/jayvee-language-server/test';
+import { AstNode, AstNodeLocator, LangiumDocument } from 'langium';
+import { NodeFileSystem } from 'langium/node';
+
+import { StdLangExtension } from './extension';
+
+describe('Validation of builtin examples of BlockMetaInformation', () => {
+  let services: JayveeServices;
+  let parse: (
+    input: string,
+    options?: ParseHelperOptions,
+  ) => Promise<LangiumDocument<AstNode>>;
+  let validate: (input: string) => Promise<ValidationResult<AstNode>>;
+
+  let locator: AstNodeLocator;
+
+  const testExtension = new TestLangExtension();
+  const stdExtension = new StdLangExtension();
+
+  async function generateFullJvExample(
```

Comment:
Can we pull this out from the `describe` to the bottom of the file?

Reply:
Only if I extend the parameters passed to the function to also include most of the in `beforeAll` initialized variables.
``` js
async function generateFullJvExample(
  blockMetaInf: BlockMetaInformation,
  blockExample: string,
): Promise<string> {}
```
would become
``` js
async function generateFullJvExample(
  blockMetaInf: BlockMetaInformation,
  blockExample: string,
  parse: (
    input: string,
    options?: ParseHelperOptions,
  ) => Promise<LangiumDocument<AstNode>>,
  locator: AstNodeLocator,
  testExtension: TestLangExtension,
): Promise<string> {}
```

Reply:
Ah well, then I'd just leave it.